### DISCUSSION TITLE
Not seeing bindable values

### DISCUSSION THREAD

**Discussion by ndlang** [Budibase/budibase#5641] [Help] [answered]
I have two option pickers that I should be able to use as inputs to my data provider but they don't appear to be visible:
![image](https://user-images.githubusercontent.com/17033357/165827611-a82c701f-45e9-4d7d-a144-b6acf73f1297.png)

**Comment by melohagan** [comment, marked as answer]
Hi @ndlang 

Your **dp_by_course** component needs to be nested within the **form_qry_params** if you want to access the form bindings from there.

**Reply by ndlang** [reply]
Thank you @melohagan. I could have sworn I tried that but I guess it was not nested with in it. Thanks again for the help.
